sympy/matrices/matrices.py

```diff
@@ -2388,7 +2388,6 @@ def dot(self, b):
         multiply_elementwise
         """
         from .dense import Matrix
-
         if not isinstance(b, MatrixBase):
```

Comment:
If `b` is a `list` or a `tuple`, doesn't it get caught by this?

Reply:
you are right , i didn't take care  of that thanks @ylemkimon